### PR TITLE
added ssl/https

### PR DESCRIPTION
**Pull request by dirkk0**

### CODE REVIEW

**Comment by vincentfretin** [comment]
Thanks @dirkk0 
related to #169
We should include mention of mkcert to create the certificate I think. And once I close #240 maybe some commented lines in the server file.

File: README.md [hunk 1]
```diff
@@ -408,6 +408,23 @@ NAF.options.useLerp
 
 By default when an entity is created the [`aframe-lerp-component`](https://github.com/haydenjameslee/aframe-lerp-component) is attached to smooth out position and rotation network updates. Set this to false if you don't want the lerp component to be attached on creation.
 
+### SSL/HTTPS
+
+To serve the files via https (because audio otherwise will silently fail due to security reasons), you have enable https to provide a secure connection.
```
Comment (vincentfretin) [inline README.md:413]: " you have enable" => " you have to enable"

File: README.md [hunk 2]
```diff
@@ -408,6 +408,23 @@ NAF.options.useLerp
...
+
+To acchieve this one needs to insert:
```
Comment (vincentfretin) [inline README.md:415]: acchieve => achieve

**Comment by vincentfretin** [comment]
Instead of the README, we should move that to https://github.com/networked-aframe/networked-aframe/blob/master/docs/hosting-networked-aframe-on-a-server.md#webrtc-audio-requires-a-ssl-certificate

**Comment by dirkk0** [comment]
Good point. I'll make the PR.

**Pull request closed without merging**